### PR TITLE
Added basic support for vancouver style

### PR DESCRIPTION
**Pull request by khwolf**
Vancouver style is often used in medicine and defined here:
Patrias K. Citing medicine: the NLM style guide for authors, editors,
and publishers [Internet]. 2nd ed. Wendling DL, technical editor.
Bethesda (MD): National Library of Medicine (US); 2007 -
[updated 2011 Sep 15; cited 2015 April 18].
Available from: http://www.nlm.nih.gov/citingmedicine

### CODE REVIEW

File: bibtexbrowser.php [hunk 1]
```diff
@@ -134,6 +134,7 @@ function bibtexbrowser_configure($key, $value) {
 @define('BIBTEXBROWSER_DEBUG',false);
 
 @define('COMMA_NAMES',false);// do have authors in a comma separated form?
+@define('INITIAL_NAMES',false); // use only initials for all first names
```
Comment (monperrus) [inline bibtexbrowser.php]: I would prefer USE_INITIALS_FOR_NAMES

**Comment by khwolf** [comment]
@monperrus i have changed that. Maybe COMMA_NAMES should be changed as well?

**Comment by monperrus** [comment]
It's a good idea. Probable USE_COMMA_AS_NAME_SEPARATOR_IN_OUTPUT is better. Could you please squash your commits in a single one afterwards?

**Comment by khwolf** [comment]
@monperrus done

**Comment by monperrus** [comment]
Thanks! --Martin

**Pull request merged**